judgels-frontends/raphael/src/components/ProblemWorksheetCard/Programming/ProblemStatementCard/ProblemStatementCard.tsx

```diff
@@ -7,6 +7,7 @@ import { ProblemStatement } from '../../../../modules/api/sandalphon/problem';
 import { ProblemLimits } from '../../../../modules/api/sandalphon/problemProgramming';
 
 import './ProblemStatementCard.css';
+import { MathjaxWrapper } from '../../../MathjaxWrapper/MathjaxWrapper';
```

Comment:
Put this directly below line 7 (convention, css import comes last)

Reply:
Okay, changed.